code/game/mecha/equipment/tools/mining_tools.dm

```diff
@@ -112,7 +112,7 @@
 	desc = "Equipment for engineering and combat exosuits. It will automatically check surrounding rock for useful minerals."
 	icon_state = "mecha_analyzer"
 	origin_tech = "materials=3;engineering=2"
-	equip_cooldown = 30
```

Comment:
AHEM: https://github.com/Core0verload/-tg-station/blob/d9a95b1e4a370c24f5f0261ec87240ebba962a0d/code/game/mecha/equipment/tools/mining_tools.dm#L146

We do not want an equip_cooldown = 0 for the mining_scanner!!!!!